test/extended/builds/pipeline.go

```diff
@@ -776,6 +780,62 @@ var _ = g.Describe("[Feature:Builds][Slow] openshift pipeline build", func() {
 				o.Expect(err).NotTo(o.HaveOccurred())
 			})
 
+			g.By("Pipeline with env vars and git repo source")
+
+			g.By("should should propagate env vars to bc", func() {
```

Comment:
`s/By/It/`  and drop the second `should`

Reply:
looks like I may need to follow the pattern of other tests here and use `By` instead of `It`, getting `You may only call It from within a Describe or Context` error